tox.ini

```diff
@@ -1,9 +1,14 @@
 [tox]
-envlist = py36,py37
+minversion = 3.23.0
+isolated_build = True
+envlist = py{36,37,38,39},flake8
```

Comment:
does this run flake8? if so do we really want tox to run flake8?

Reply:
It does! And I think we should do both, for the same reason that we run it in both pre-commit and in CI. The more places, the better. Not all contributors will install `pre-commit` in their local environment, and not all contributors will run `tox`. But the more places we run these tools, the higher chance that they will get run at least once by contributors.